File: Chapter2/2.3/problem5.py
x = int(input("Enter Your X Value:"))
y = int(input("Enter Your Y Value:"))

if x>0 and y>0:
    print("First Quadrant")
elif(x<0 and y>0):
    print("Second Quadrant")
elif(x<0 and y<0):
    print("Third Quadrant")
elif(x>0 and y<0):
    print("Four Quadrant")

File: Chapter2/2.3/problem5.c
#include<stdio.h>

int main(){
    int x,y;

    printf("Enter Your X Value: ");
    scanf("%d", &x);

    printf("Enter Your Y Value: ");
    scanf("%d", &y);

    if(x>0 && y>0){
        printf("First Quadrant");
    }
    else if(x<0 && y>0){
        printf("Second Quadrant");
    }
    else if(x<0 && y<0){
        printf("Third Quadrant");
    }
    else if(x>0 && y<0){
        printf("Four Quadrant");
    }

    

}